pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
     "osc",
     "openqa-client",
     "pika",
+    "pyzstd",
```

Comment:
this package does not seem to be available on the current Leap 15.6 based system. Can we consider alternatives? Otherwise wait for https://progress.opensuse.org/issues/191617